Documentation: Add clarity to deployment options (#20914)
fleetdm/fleet: docs/Deploy/deploy-fleet.md

# Deploy Fleet

> Just need to kick the tires? Create an account to [Try Fleet locally](https://fleetdm.com/try-fleet) on your device. You can deploy FleetDM on your own servers or virtual machines, which involves manually setting up the necessary components, such as the FleetDM server, database, and any dependencies. This approach, although more lengthy than a Cloud deployment, gives you full control over the environment and configuration.
> A technical installation guide sponsored by Internews for a self-contained single server deployment of FleetDM on Ubuntu, with a few Elastic Stack components, can be found [here](https://fleetdm.com/guides/deploy-fleet-on-ubuntu-with-elastic).
> 
> On the other hand, use our easy steps below to deploy to Render for an easy one-click proof of concept. Or, choose AWS with Terraform to deploy at scale.

Reviewer: Let's keep this intro dead simple. We can add a note linking to the guides below instead.

```suggestion
Deploy to Render for an easy one-click proof of concept. Or, choose AWS with Terraform to deploy at scale. Just need to kick the tires? [Try Fleet locally](https://fleetdm.com/try-fleet) on your device.
```

    terraform apply
    ```

> 12. That’s it! Next, you can skip ahead and already log in to Fleet to [enroll a host](https://fleetdm.com/docs/using-fleet/enroll-hosts), or you can continue with our other Deployment documents for other options that might fit your environment:
> - [Single sign-on (SSO)](https://fleetdm.com/docs/deploy/single-sign-on-sso)
> - [Upgrading Fleet](https://fleetdm.com/docs/deploy/upgrading-fleet)
> - [Reference architectures](https://fleetdm.com/docs/deploy/reference-architectures)

Reviewer: Let's revert this change. We already have all these links in the deployment section of the side nav, and introducing duplication here makes it harder to keep these up-to-date.
```suggestion
12. That’s it! You should now be able to log in to Fleet and [enroll a host](https://fleetdm.com/docs/using-fleet/enroll-hosts).
```
